src/post.js

```diff
@@ -70,11 +70,7 @@ const Post = ({ back = '/blog', children, meta, number, id, ...props }) => {
   })
   const notProduction = process.env.VERCEL_ENV !== 'production'
   const baseUrl = notProduction ? '' : 'https://blog.carbonplan.org'
-  const cardUrl = meta.card
```

Comment:
Hm why get rid of this check? It might nice to leave this around for a while just to be safe, but also _especially_ to avoid an unnecessary major version bump (if we can also restore/support the `forceWrapAuthors` naming convention).